tests/Agent/UnitTests/NewRelic.Agent.Extensions.Tests/Reflection/VisibilityBypasserTests.cs

```diff
@@ -552,5 +552,63 @@ public void no_constructor_found()
         {
             Assert.Throws<Exception>(() => VisibilityBypasser.Instance.GenerateTypeFactory<bool, int, PublicOuter>());
         }
+
+        [Test]
```

Comment:
I'm curious how this new test is related to this PR.

Reply:
My bad, I should have put that in the description. It's just more code coverage for a class that was high on the list.